scene/resources/audio_stream_wav.h

```diff
@@ -138,6 +139,8 @@ class AudioStreamWAV : public AudioStream {
 	LocalVector<uint8_t> data;
 	uint32_t data_bytes = 0;
 
+	static Ref<AudioStreamWAV> load_from_dr_wav(drwav &p_wav, const Dictionary &p_options);
```

Comment:
I'm not sure about exposing the thirdparty publically like that. I think we should define our own options.

Reply:
It's supposed to be a private function only called by both `load_from_file()` and `load_from_buffer()` after gathering all metadata and generating the `drwav` struct.

Before #93831, the entire WAV file reading step used a `FileAccess` to read from the disk, but the PR changed it to load the entire file data into memory before reading. I wanted to take advantage with this PR to make `load_from_file()` actually read from the disk, but didn't see a way to make the code smaller without slicing the function, which resulted into `load_from_dr_wav()` being created.

Reply:
I would stop at load from wav and we consider the type of library a implementation detail that must be hidden.

Reply:
After a rework, this function doesn't exist anymore, so I'm marking this as resolved.